source/refills/tooltipped.html.erb

```diff
@@ -0,0 +1,7 @@
+<a href="#0" class="tooltipped" aria-label="View Notifications">
+  <svg aria-hidden="true" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
I think it would be useful to explain that this tooltip is built for a graphic-only button/link, in which the tooltip is the only and primary text describing its action.